setup/docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Pull Maven with JDK 7 for build time docker
+FROM maven:3.5.4-jdk-7-slim AS build
```

Comment:
Maybe build the project on JDK 8 if is using runtime machine?
[maven:3.5.4-jdk-8-alpine](https://hub.docker.com/_/maven/)